app/models/bank_account.rb

```diff
@@ -24,4 +24,55 @@ def assign_unlinked_transactions
     end
     count
   end
+
+  def last_transaction_date
+    bank_transactions.order(date: :desc).first.try(:date)
+  end
+
+  def import_account_information!(content)
+    return nil if content.empty?
+    data = JSON.parse content, symbolize_names: true
+
+    booked = data.try(:fetch, :transactions, nil).try(:fetch, :booked, nil) || []
```

Comment:
Do you agree that this use of `try` reduces readability?
What about:
```
return nil if data.empty?

booked = data.fetch(:transactions, {}).fetch(:booked, [])
```

Reply:
agree